doc/data/messages/d/dict-init-mutate/bad.py

```diff
@@ -0,0 +1,3 @@
+fruits = {}  # [dict-init-mutate]
```

Comment:
I suggest sticking with the `config` example here instead of fruits, so as to emphasize the real-world existence of this pattern.

Reply:
HA! @Pierre-Sassoulas  loves fruits ( I think?) so I just stick to them since he approves :)

Reply:
I'm using fruit and animals because I'm unimaginative but any example that a school children would understand works 😄 just no ``foo``, ``bar``, or ``baz``.